nexus/db-queries/src/db/datastore/address_lot.rs

```diff
@@ -225,6 +278,35 @@ impl DataStore {
 
         Ok(address_lot_id)
     }
+
+    // Take the name of an address lot and look up its blocks
+    pub async fn address_lot_blocks_by_name(
+        &self,
+        opctx: &OpContext,
+        name: String,
+    ) -> LookupResult<Vec<AddressLotBlock>> {
+        let conn = self.pool_connection_authorized(opctx).await?;
+
+        use db::schema::address_lot::dsl as lot_dsl;
+        use db::schema::address_lot_block::dsl as block_dsl;
+
+        let address_lot_id = lot_dsl::address_lot
```

Comment:
nitpick: This could be a single query with a join, but I think this is also fine as-is